dataland-frontend/src/components/resources/dataTable/conversion/Utils.ts

```diff
@@ -66,6 +67,15 @@ export function getDataPointGetterFactory<
       displayValue = formattedValue;
     }
     const dataPointAsExtendedDataPoint = dataPoint as unknown as ExtendedDataPoint<V>;
+    if (
+      dataPointAsExtendedDataPoint.quality == QualityOptions.Na &&
```

Comment:
What if a comment is provided?

Reply:
Added a condition that the comment must also be null